resources/language/resource.language.nl_nl/strings.po

```diff
@@ -0,0 +1,166 @@
+# Service Up Next language file
+# Addon Name: Up Next
+# Addon id: service.upnext
+# Addon Provider: im85288
+msgid ""
+msgstr ""
+"Project-Id-Version: service.upnext\n"
+"Report-Msgid-Bugs-To: http://trac.xbmc.org/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2018-02-26 13:31+0000\n"
+"Last-Translator: dagwieers\n"
+"Language-Team: Dutch\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7\n"
+
+msgctxt "#30001"
+msgid "The number of seconds before the end to show the notification"
+msgstr "Aantal seconden voor het einde om meldingen te tonen"
+
+msgctxt "#30002"
+msgid "Log Level:"
+msgstr "Log niveau"
+
+msgctxt "#30003"
+msgid "Settings"
+msgstr "Instellingen"
+
+msgctxt "#30004"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "#30005"
+msgid "Default action when nothing selected"
+msgstr "Standaard actie als niets geselecteerd is"
+
```

Comment:
Standaardactie is één woord.

Reply:
Ja, dat doe ik dus iedere keer verkeerd. Ik had er nu nog speciaal op gelet.
En toch...

Reply:
Fixed.

Reply:
In het Nederlands wordt heel veel aaneengeschreven: https://www.taaltelefoon.be/aaneenschrijven-01-hoofdregels